openwillis/measures/video/facial_emotion.py

```diff
@@ -57,7 +58,7 @@ def run_deepface(path, measures):
                 img_rgb = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
 
                 face_analysis = DeepFace.analyze(img_path = img_rgb, actions = ['emotion'])
-                df_face = pd.DataFrame([face_analysis['emotion'].values()], columns=cols)/100
+                df_face = pd.DataFrame([face_analysis['emotion'][0].values()], columns=cols)/100
```

Comment:
So in my tests, it looks like this should be as following. Do you think it should be as is? If that's the case we should look into possible package version mismatch

```suggestion
                df_face = pd.DataFrame([face_analysis[0]['emotion'].values()], columns=cols)/100
```